src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.scss

```diff
@@ -87,21 +93,12 @@ $shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
   }
 
   .LogoContainer {
-    display: none;
+    border-right: none;
   }
 
-  .Contents {
-    margin: 0;
-    padding: 0 spacing();
-    max-width: 100%;
-
-    @include page-content-when-not-fully-condensed {
-      padding: 0 spacing();
-    }
-
-    @include page-content-when-not-partially-condensed {
-      padding: 0 spacing();
-    }
+  .ContextControl {
+    opacity: 0.3;
+    // pointer-events: none;
```

Comment:
🔥

Reply:
Whoops, we need this but I had it commented out to check paddings. Need to add it back